egs/fisher_callhome_es-en/asr1/run.sh

```diff
@@ -0,0 +1,373 @@
+#!/bin/bash
+set -e
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch
+stage=0        # start from 0 if you need to start from data preparation
+gpu=0         # use 0 when using GPU on slurm/grid engine, otherwise -1
```

Comment:
Use multiple GPU options by referring the other recipes. Search `ngpu` to find the place that you have to change.